src/app/modules/twenty-four-hour-time/twenty-four-hour-time.component.css

```diff
@@ -0,0 +1,203 @@
+.ds-c-field {
+  margin-bottom: 0;
+  margin-top:5px;
+}
+
+.ds-c-label {
+  margin-top: 0;
+}
+
+.time-container {
+  margin-left: 0px;
+  position: relative;
+  border-radius: 3px;
+  padding: 8px 0px !important;
+}
+
+.time-container-focus {
+  outline-width: 2px;
+  outline-style: solid;
+  outline-color: Highlight;
+}
+
+.time-container-border {
+  border: 2px solid #323a45 !important;
+}
+
+.time-container-error-border {
+  border: 3px solid #e31c3d !important;
+}
+
+
+.time-container-error-shake {
+  animation: shake 0.2s ease-in-out 0s 2;
+  box-shadow: 0 0 0.5em #e31c3d;
+}
+
+@keyframes shake {
+  0% { margin-left: 0rem; }
+  25% { margin-left: 0.5rem; }
+  75% { margin-left: -0.5rem; }
+  100% { margin-left: 0rem; }
+}
+
+.time-input {
+  border: none;
+  width: 100%;
+}
+
+.time-input:focus {
+  outline: none;
+}
+
+.time-input:disabled {
+  background: lightgrey;
+}
+
+.time-picker {
+  position: absolute;
+  background: white;
+  z-index: 1;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 13px 27px -5px, rgba(0, 0, 0, 0.3) 0px 8px 16px -8px;
+}
+
+.time-picker-label {
+  flex-grow: 1;
+}
+
+.time-picker-header {
+  display: flex;
+  justify-content: space-between;
+  font-weight: bold;
```

Comment:
In my `pillbox` [PR](https://github.com/Bellese/angular-design-system/pull/354/files), I tried to use `ds-u-...` classes from CMS design system where possible. Is this a convention we wish to continue?

`src/app/modules/pillbox/pillbox.component.html` i.e. 
 - `ds-u-padding-left--1`
 - `ds-u-font-size--base`
 - `ds-u-font-weight--bold`

Reply:
Nice catch, I'll update these